vtex-apps/store-form
# Fix lint problems + Fix tests

#### What does this PR do? \*

Add lint configuration + Fix lint problems + Fix tests + Update dependencies

#### How to test it? \*

https://checkbox--unileverb2b.myvtex.com/#
![image](https://user-images.githubusercontent.com/8517023/81598890-5e093900-939e-11ea-8a58-a4ab222a38ec.png)
Click in `Registrar`
It should run

The tests are not working yet, so I need some help guys
